lib/graphiti/debugger.rb

```diff
@@ -24,7 +26,7 @@ def on_data(name, start, stop, id, payload)
         end
       end
 
-      def on_data_exception(payload, params)
+      private def on_data_exception(payload, params)
```

Comment:
I think these methods should be private since they're only used to support the main notification callbacks. This also saves some testing 😄 

I opted for "inline" private modifier to keep diff as small as possible. If preferred I can move this methods under the private group.